ports/esp32/machine_bitstream.c

```diff
@@ -172,16 +173,20 @@ static void machine_bitstream_high_low_rmt(mp_hal_pin_obj_t pin, uint32_t *timin
     // Cancel RMT output to GPIO pin.
     esp_rom_gpio_connect_out_signal(pin, SIG_GPIO_OUT_IDX, false, false);
 }
-
+#endif
 /******************************************************************************/
 // Interface to machine.bitstream
 
 void machine_bitstream_high_low(mp_hal_pin_obj_t pin, uint32_t *timing_ns, const uint8_t *buf, size_t len) {
+    #if SOC_RMT_SUPPORTED
     if (esp32_rmt_bitstream_channel_id < 0) {
         machine_bitstream_high_low_bitbang(pin, timing_ns, buf, len);
     } else {
         machine_bitstream_high_low_rmt(pin, timing_ns, buf, len, esp32_rmt_bitstream_channel_id);
     }
+    #else
+    machine_bitstream_high_low_bitbang(pin, timing_ns, buf, len);
+    #endif
```

Comment:
I'm probably being too picky here, so feel free to ignore this suggestion, but this could be a little cleaner as:
```c
    #if SOC_RMT_SUPPORTED
    if (esp32_rmt_bitstream_channel_id >= 0) {
        machine_bitstream_high_low_rmt(pin, timing_ns, buf, len, esp32_rmt_bitstream_channel_id);
        return;
    }
    #endif
    machine_bitstream_high_low_bitbang(pin, timing_ns, buf, len);
}
```